app/helpers/precheck_helper.rb

```diff
@@ -11,8 +11,20 @@ def precheck_eligibility_actionable_errors(family)
     PrecheckEligibilityService.new(family: family).actionable_errors
   end
 
-  def precheck_eligibility_error_label(error)
-    error.to_s.humanize
+  def precheck_eligibility_error_label(error, family)
+    {
+      not_checked_in_already?: 'Already checked-in',
```

Comment:
Nitpicking, but I suppose these could be moved into locale folder, since thats where most strings used on ui go.

Reply:
Valid feedback! Thanks for encouraging me 😄 I've added another commit for that.